src/components/UserFeedbackComponent.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Container, Header, Table, Pagination } from 'semantic-ui-react';
+import feedbackAction from '../_actions/feedback.action';
+import SideMenuComponent from '../_components/SideMenuComponent';
+import TableRowComponent from './TableRowComponent';
+
+export class UserFeedbackComponent extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      activePage: 1,
+      limit: 10,
+    };
+  }
+
+  componentDidMount() {
+    this.props.feedbackAction(this.state.defaultPage);
+  }
+
+  handlePaginationChange = (e, { activePage }) => {
+    this.setState({ activePage });
+    this.props.feedbackAction(activePage);
+  }
+
+  handlePageTotal = () => Math.ceil(this.props.feedbackCount / this.state.limit)
+
+  emptyFeedbackCount = () => (this.props.feedbackCount === 0)
+
+  loadFeedback = () => {
+    if (this.emptyFeedbackCount()) {
+      return (
+        <Table.Row>
+          <Table.Cell colSpan="6">No Data found</Table.Cell>
+        </Table.Row>
+      );
+    }
+    const feedbacks = this.props.feedback.map(feedback => (
```

Comment:
You can't pluralize **feedback** either change the name of the variable to something like **feedback_records** or just call it feedback. There is no such word as feedbacks.

Reply:
Yes, I agree.. This was already added before I added my changes.